### PR TITLE
testing/lzop: new aport

### PR DESCRIPTION
**Pull request by sgerrand**
💁 An improvement on the version available in BusyBox.

### CODE REVIEW

File: testing/lzop/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Contributor: Sasha Gerrand <alpine-pkgs@sgerrand.com>
+# Maintainer: Sasha Gerrand <alpine-pkgs@sgerrand.com>
+pkgname=lzop
+pkgver=1.04
+pkgrel=0
+pkgdesc="lzop is a very fast file compressor"
+url="https://www.lzop.org/"
+arch="all"
+license="GPL2"
```
Comment (maxice8) [inline testing/lzop/APKBUILD]: this is ambiguous and not SPDX compliant

File: testing/lzop/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+depends="lzo"
+depends_dev=""
```
Comment (maxice8) [inline testing/lzop/APKBUILD]: empty variables can be removed

File: testing/lzop/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+makedepends="$depends_dev gcc lzo-dev make"
```
Comment (maxice8) [inline testing/lzop/APKBUILD]: calling gcc and make is not required and it is referencing an empty (removed) variable $depends_dev

File: testing/lzop/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+install=""
+subpackages="$pkgname-doc"
+source="https://www.lzop.org/download/lzop-$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
```
Comment (maxice8) [inline testing/lzop/APKBUILD]: default value can be removed

File: testing/lzop/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+check() {
+        cd "$builddir"
+        make check
+}
+
+package() {
+        cd "$builddir"
+        make DESTDIR="$pkgdir" install
+        install -Dm644 COPYING "$pkgdir"/usr/share/licenses/$pkgname/COPYING
```
Comment (maxice8) [inline testing/lzop/APKBUILD]: no need to install LICENSE file

File: testing/lzop/APKBUILD [hunk 6]
```diff
@@ -0,0 +1,41 @@
+# Contributor: Sasha Gerrand <alpine-pkgs@sgerrand.com>
+# Maintainer: Sasha Gerrand <alpine-pkgs@sgerrand.com>
+pkgname=lzop
+pkgver=1.04
+pkgrel=0
+pkgdesc="lzop is a very fast file compressor"
+url="https://www.lzop.org/"
+arch="all"
+license="GPL2"
+depends="lzo"
```
Comment (maxice8) [inline testing/lzop/APKBUILD]: This can problably be removed

File: testing/lzop/APKBUILD (hunk 1 above)
Comment (sgerrand) [reply testing/lzop/APKBUILD]:
```suggestion
license="GPL-2.0-only"
```

File: testing/lzop/APKBUILD (hunk 2 above)
Comment (sgerrand) [reply testing/lzop/APKBUILD]:
```suggestion
```

File: testing/lzop/APKBUILD (hunk 3 above)
Comment (sgerrand) [reply testing/lzop/APKBUILD]:
```suggestion
makedepends=""
```

File: testing/lzop/APKBUILD (hunk 4 above)
Comment (sgerrand) [reply testing/lzop/APKBUILD]:
```suggestion
```

File: testing/lzop/APKBUILD (hunk 5 above)
Comment (sgerrand) [reply testing/lzop/APKBUILD]:
```suggestion
```

File: testing/lzop/APKBUILD (hunk 6 above)
Comment (sgerrand) [reply testing/lzop/APKBUILD]: Yep, `abuild` figures it out.
```suggestion
```

File: testing/lzop/APKBUILD [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+# Maintainer: Sasha Gerrand <alpine-pkgs@sgerrand.com>
+pkgname=lzop
+pkgver=1.04
+pkgrel=0
+pkgdesc="lzop is a very fast file compressor"
+url="https://www.lzop.org/"
+arch="all"
+license="GPL-2.0-only"
+makedepends="lzo-dev"
+install=""
```
Comment (maxice8) [inline testing/lzop/APKBUILD]: empty variables can be removed

**Comment by maxice8** [comment]
Squash everything into a single commit

**Comment by sgerrand** [comment]
@maxice8: Done. ♻️

**Pull request closed without merging**